neutrino.go

```diff
@@ -620,6 +628,37 @@ type Config struct {
 	BroadcastTimeout time.Duration
 }
 
+// HeadersImportConfig contains configuration options for importing headers
+// from external sources.
+type HeadersImportConfig struct {
+	// BlockHeadersSource specifies where to obtain block headers from.
+	// This could be a file path, URL, or other source identifier.
+	BlockHeadersSource string
+
+	// FilterHeadersSource specifies where to obtain filter headers from.
+	// This could be a file path, URL, or other source identifier.
+	FilterHeadersSource string
+
+	// WriteBatchSizePerRegion defines the number of headers to write in a
+	// single batch per processing region during import. The import process
+	// divides header ranges into distinct regions
+	// (overlap, divergence, beyond import, and new headers), each with
+	// different processing requirements. Each region is processed in
+	// batches of this size to optimize database performance while
+	// maintaining data integrity boundaries. Larger values improve speed
+	// but increase memory usage.
+	//
+	// Default value: 16,384 (2^14) entries.
```

Comment:
Should we actually make this the default value? Meaning that if the value is given as `0` in `ChainService.Start()` we assign the 16k default value?